LabProject/src/main/java/agh/ics/oop/MapBoundaryElement.java

```diff
@@ -0,0 +1,11 @@
+package agh.ics.oop;
+
+public class MapBoundaryElement {
```

Comment:
fajny pomysł, ale pokazywaliśmy już jak robić rekordy - to jest idealny przykład rekordu ;)